cwms-data-api/src/main/java/cwms/cda/data/dto/forecast/ForecastSpec.java

```diff
@@ -0,0 +1,78 @@
+package cwms.cda.data.dto.forecast;
+
+import cwms.cda.api.errors.FieldException;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import javax.xml.bind.annotation.*;
+import java.util.List;
+
+@XmlRootElement(name = "forecast-spec")
+@XmlAccessorType(XmlAccessType.FIELD)
+public class ForecastSpec {
+
+  @Schema(description = "Forecast Spec ID")
+  @XmlElement(name = "spec-id")
+  String specId;
+
+  @Schema(description = "Office ID that owns the time-series")
+  @XmlElement(name = "office-id")
+  String officeId;
+
+  @Schema(description = "Location ID")
+  @XmlElement(name = "location-id")
+  String locationId;
+
+  @Schema(description = "Source Entity ID")
+  @XmlElement(name = "source-entity-id")
+  String sourceEntityId;
+
+  @Schema(description = "Description of Forecast")
+  @XmlAttribute
+  private String description;
+
+  @Schema(description = "List of Time Series IDs belonging to this Forecast Spec")
+  @XmlAttribute(name = "time-series-ids")
+  private List<String> timeSeriesIds;

```

Comment:
should be list of time series identifiers. We'll likely need to meet to talk about this

Reply:
instead of `List<String>` use `TimeSeriesIdentifierDescriptors`

Reply:
Looks like the object mapper readValue function throws an error after this update:

com.fasterxml.jackson.databind.exc.InvalidDefinitionException: Cannot construct instance of `cwms.cda.data.dto.TimeSeriesIdentifierDescriptors$Builder` (no Creators, like default constructor, exist): cannot deserialize from Object value (no delegate- or property-based Creator)

 writeValueAsString works though.

Reply:
if @rripken is still around maybe he knows a tip? otherwise I can help take a look on Monday

Reply:
I'm guessing its just the wrong class.  Use a List<TimeSeriesIdentifierDescriptor> the TimeSeriesIdentifierDescriptors is for something else.

Reply:
Unit tests are passing after changing to a List!